tensorflow/lite/micro/tools/make/ext_libs/cmsis_nn.inc

```diff
@@ -14,10 +20,12 @@ ifeq ($(CMSIS_PATH), $(CMSIS_DEFAULT_DOWNLOAD_PATH))
   endif
 endif
 
-THIRD_PARTY_CC_SRCS += \
-  $(call recursive_find,$(CMSIS_PATH)/CMSIS/NN/Source,*.c)
-THIRD_PARTY_CC_HDRS += \
-  $(call recursive_find,$(CMSIS_PATH)/CMSIS/NN/Include,*.h)
+ifeq (,$(CMSIS_NN_LIBS))
+  THIRD_PARTY_CC_SRCS += $(call recursive_find,$(CMSIS_PATH)/CMSIS/NN/Source,*.c)
```

Comment:
What do you think of replace this with a find shell command (and then we can replace the recursive_find macro and have the makefiles be a bit simpler)?

something like
https://github.com/tensorflow/tflite-micro/blob/de29703ba0847677ce7cc7df1165e222501946da/tensorflow/lite/micro/tools/make/ext_libs/xtensa.inc#L30-L31

Reply:
Will that work on Window? I recall something like that was the reason for recursive_find. If yes I am fine with replacing it.

Reply:
Unclear. We are already using various find commands, so I think we can switch this one over as well.

https://github.com/tensorflow/tflite-micro/blob/96e109101e60737e17115d16660591c38b90d69d/tensorflow/lite/micro/tools/make/Makefile#L386-L390

With the new project generation and #407, we expect the majority of Windows use-cases (which were around makefile-based project generation) to be supported via external TFLM examples repositories.

We do have users attempting to `make` on Windows and that is failing for reasons that we have been open about not being able to support (e.g. https://github.com/tensorflow/tflite-micro/issues/170)